### DISCUSSION TITLE
The menu option of responsive navbar

### DISCUSSION THREAD

**Discussion by taseenjeem** [saadeghi/daisyui#1634] [Q&A] [answered]
In the responsive version of the navbar, if I open the menu option and change the route, the menu option still remains open. How can I turn it off after click?

**Comment by saadeghi** [comment, marked as answer]
Currently dropdown works with CSS focus. So as long as it is focused, it will stay open. You can remove the focus in many ways depending on what JS library you're using. Read this discussion for more info: https://github.com/saadeghi/daisyui/issues/201

Let me know if you have any questions.
